robotiq_driver/src/gripper_interface_test.cpp

```diff
@@ -35,58 +35,61 @@ constexpr auto kComPort = "/dev/ttyUSB0";
 constexpr auto kSlaveID = 0x09;
 
 int main() {
-    RobotiqGripperInterface gripper(kComPort, kSlaveID);
+    try{
```

Comment:
```suggestion
    try {
```

Same with all the other try blocks in this PR